src/SpeedyTransforms/spectral_transform.jl

```diff
@@ -27,12 +26,12 @@ struct SpectralTransform{
     mmax::Int                       # Maximum order m=[0, l] of spherical harmonics
     nfreq_max::Int                  # Maximum (at Equator) number of Fourier frequencies (real FFT)
     LegendreShortcut::Type{<:AbstractLegendreShortcut} # Legendre shortcut for truncation of m loop
-    mmax_truncation::VectorIntType  # Maximum order m to retain per latitude ring
+    mmax_truncation::Vector{Int}   # Maximum order m to retain per latitude ring
```

Comment:
@maximilian-gelbrecht vectors of integers that don't actually contain any data, in this case just defines the end of a range for a loop, aren't supposed to live on the GPU, right?

Reply:
Indices don't have to be stored on the GPU.